### ISSUE TITLE
Podspec is incompatible w/ Xcode 9 "preview build" option

### ISSUE THREAD

**Issue by wildthink** [bustoutsolutions/siesta#242]
The new Xcode 9 "Project/Workspace Settings" -> "Build System" "New Build System (Preview)" option is stricter with the Xcode configuration and produces an error for projects including Siesta.

```
error: unexpected duplicate task: CompileXIB xxxx/Pods/Siesta/Source/SiestaUI/ResourceStatusOverlay.xib (in target 'Siesta')
```

The fix is as follows. Note by explicitly specifying the source file extensions, it ignores xibs which are included as resource.
```
s.subspec "Core" do |s|
    s.source_files = "Source/Siesta/**/*.swift"
    s.exclude_files = "**/Info*.plist"
  end

  s.subspec "UI" do |s|
    s.ios.source_files = "Source/SiestaUI/**/*.swift"
    s.dependency "Siesta/Core"
    s.exclude_files = "**/Info*.plist"
    s.ios.resources = "Source/**/*.xib"
  end
```

**Comment by pcantrell** [comment]
Thanks for flagging this! The fix should go out in the next release.

**Comment by pcantrell** [comment]
Couldn’t do it all without the help of people like you!

I’m going to leave this open for now just so I don’t forget to actually roll out the fix. Because I know my own brain, and that will totally happen.

**Comment by mglass** [comment]
@pcantrell The new build system is now the default in Xcode 10. Can we expect this fix (and incidentally, the fix for #244) anytime soon? Might be worth a bug fix release at this stage if 1.4 isn't ready.

**Comment by NermeenNery** [comment]
how can add this in pod file ? any example ??

s.subspec "Core" do |s|
    s.source_files = "Source/Siesta/**/*.swift"
    s.exclude_files = "**/Info*.plist"
  end

  s.subspec "UI" do |s|
    s.ios.source_files = "Source/SiestaUI/**/*.swift"
    s.dependency "Siesta/Core"
    s.exclude_files = "**/Info*.plist"
    s.ios.resources = "Source/**/*.xib"
  end

**Comment by pcantrell** [comment]
@NermeenNery: You shouldn’t need to modify any podfiles; this fix is already released, and is included in Siesta version 1.4.0 and larger.

If you are getting an error that sounds like this issue, you should make sure that you are using the latest version of Siesta. Take some time to understand how Podfiles specify versions, how you can use `Podfile.lock` to see what version of each pod you have installed, and how to update a pod version if necessary.

If you see that you are using the latest version of Siesta but still getting this error, then create an issue that includes enough information for others to reproduce the problem. The [Mozilla guide to bug reports](
https://developer.mozilla.org/en-US/docs/Mozilla/QA/Bug_writing_guidelines) I linked to in your earlier issue is  good resource.
